### ISSUE TITLE
Snapdrop Brave for Android with Snapdrop Brave for Windows

### ISSUE THREAD

**Issue by dtantono** [RobinLinus/snapdrop#339]
Hi
I want to send a message and jpeg file between Android and Windows vice versa. But, it doesn't worked. Is it a bug?

**Comment by Bellisario** [comment]
Yes, this could be a known Snapdrop bug, but for now there isn't any code change that works (it's a problem of WebRTC).
For now, you could try to look at these issues if somethings resolves your problem: #322 (have a look also on linked issues).

**Comment by dtantono1** [comment]
because Android and Windows have market share more than 50 % user base globally and mobility is quite important nowadays. That means this problem have a major impacts. So, I hope this problem will be solved soon.
